src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/RedisExtIntegrationTest.php

```diff
@@ -254,15 +253,15 @@ public function testGetNonBlocking()
     public function testGetAfterReject()
     {
         $redis = new \Redis();
-        $connection = Connection::fromDsn('redis://localhost/messenger-rejectthenget', ['delete_after_ack' => true], $redis);
+        $connection = Connection::fromDsn('redis://localhost/messenger-rejectthenget', ['delete_after_ack' => true, 'sentinel_master' => getenv('MESSENGER_REDIS_SENTINEL_MASTER')], $redis);
 
         $connection->add('1', []);
         $connection->add('2', []);
 
         $failing = $connection->get();
         $connection->reject($failing['id']);
 
-        $connection = Connection::fromDsn('redis://localhost/messenger-rejectthenget', ['delete_after_ack' => true]);
+        $connection = Connection::fromDsn('redis://localhost/messenger-rejectthenget', ['delete_after_ack' => true, 'sentinel_master' => getenv('MESSENGER_REDIS_SENTINEL_MASTER')]);
```

Comment:
this is going to be `false` when the env var doesn't exist, but looking at the code, it compares against `null`, so that passing `false` can lead to unexpected behavior isn't it?